### ISSUE TITLE
UI Artefacting

### ISSUE THREAD

**Issue by Wffv9FNa** [dreautall/waterfly-iii#406]
# Description
Since the update released June 1st there is significant artefacting throughout the app making various icons and text illegible.

# Steps to reproduce
Issue occurs immediately on opening the app

# Expected Behaviour
All UI elements visible and legible

# Actual Behaviour
Various UI elements affected by green artefacting

# Screenshots
![Screenshot_20240607_155534](https://github.com/dreautall/waterfly-iii/assets/63041557/e300e706-5194-4666-a166-912da20f3b09)
![Screenshot_20240607_155548](https://github.com/dreautall/waterfly-iii/assets/63041557/5e73c35f-1c78-4900-b811-58c2bf02914c)
![Screenshot_20240607_155622](https://github.com/dreautall/waterfly-iii/assets/63041557/67ecd33e-ee51-4d4c-8fc8-80a5fa687204)

# OS
Samsung Galaxy S22 Ultra
Samsung Exynos 2200
GPU: ANGLE (Samsung Xclipse 920) on Vulkan 1.1.179
RAM size: 12 GB
Size: 256 GB
Android Version: Android 13 (Tiramisu)
S908BXXU6CWH5
Security patch: 1 Aug 2023

**Comment by tippfehlr** [comment]
had this on startup, the bottom half of the app was in green like a broken screen (rhyme unintended)
cant reproduce this though 🤷‍♂️

**Comment by Wffv9FNa** [comment]
> had this on startup, the bottom half of the app was in green like a broken screen (rhyme unintended) cant reproduce this though 🤷‍♂️

Did you take any steps to resolve it or did it just sort itself out? 
For me the app is always like this now both on light and dark theme with dynamic colours on and off.

**Comment by tippfehlr** [comment]
No, I did nothing, it was gone after the loading screen and didn’t
reappear.

Sorry :)
